src/app/styles.tsx

```diff
@@ -0,0 +1,145 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { openSans } from '@/styles/fonts';
+
+export const PageContainer = styled.div`
+  width: 100%;
+  min-height: 100vh;
+`;
+
+export const TitleSection = styled.div`
+  position: relative;
+  width: 100%;
+  min-height: 100vh; // Use 100vh to make the div fill the entire viewport height
+  display: flex;
+  flex-direction: column;
+  overflow-y: hidden;
+`;
+
+export const ButtonDiv = styled.div`
+  margin-top: 25px;
+  width: clamp(224px, 25%, 500px);
+  height: clamp(40px, 5vh, 60px);
+`;
+
+export const StatisticsSection = styled.div`
+  width: 100%;
+  height: 146;
+  background-color: white;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  box-shadow: 0px 8px 20px rgba(0, 0, 0, 0.1);
+  padding-top: 33px;
+  padding-bottom: 35px;
+  margin-bottom: 20px;
+`;
+
+// Label for each stat
+export const StatLabel = styled.div`
+  font-size: 1.375rem;
+  color: ${COLORS.greyDark};
+`;
+export const StatContainer = styled.div`
+  text-align: center;
+`;
+export const HowYouCanHelpContainer = styled.div`
+  background-color: ${COLORS.background};
+  width: 100%;
+  min-height: 557px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  padding-top: 70px;
+`;
+
+// Service container
+export const ServicesDiv = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-top: 60px;
+  gap: 90px;
+  padding-left: 190px;
+  padding-right: 190px;
+`;
+export const ServiceContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  text-align: center;
+  margin: 10px;
+`;
+
+// Service icon
+export const ServiceIcon = styled.img`
+  height: 50px;
+  margin-bottom: 10px;
+`;
+
+export const MissionValuesContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 2fr; // Example adjustment
+  grid-template-rows: auto;
+  gap: 20px; // Adjust space between grid items
+  background-color: ${COLORS.blueLighter};
+  width: 100%;
+  min-height: 488px;
+  align-items: center;
+  justify-content: space-around;
+  gap: 50px;
+`;
+export const MissionStatement = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  max-width: 500px;
+  gap: 10px;
+  padding-left: 50px;
+`;
+
+export const TextContainer = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  max-width: 900px;
+  padding-top: 200px;
+  padding-left: 84px;
+  padding-bottom: 150px;
+  z-index: 1;
+  padding-right: 32px;
+  gap: 50px;
+  flex-wrap: wrap;
```

Comment:
Let's use `gap: 30px` to match Figma.
We don't need `padding-right` here, so we can get rid of it.
For `max-width` and the rest of the paddings, let's add some clamping with reasonable values to address for mobile view.

If you need to play around with how it looks like on mobile, open up the web tool (F12 or right click and inspect element) and either resize the toolbar so it's close to mobile width (~400px) or click on the icon that looks like a laptop and phone to adjust the device size.